### PR TITLE
Replace prioritized_replay_buffer as PrioritizedBufferWrapper

### PR DESCRIPTION
**Pull request by jinPrelude**
replace prioritized_replay_buffer as PrioritizedBufferWrapper.

1. prioritized_replay_buffer -> PrioritizedBufferWrapper
2. buffer_size -> max_len

### CODE REVIEW

File: rl_algorithms/common/buffer/wrapper.py [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
-from rl_algorithms.common.buffer.replay_buffer import ReplayBuffer
+from rl_algorithms.common.abstract.buffer import BaseBuffer, BufferWrapper
 from rl_algorithms.common.buffer.segment_tree import MinSegmentTree, SumSegmentTree
 
 device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
 
 
-class PrioritizedReplayBuffer(ReplayBuffer):
-    """Create Prioritized Replay buffer.
+class PERWrapper(BufferWrapper):
```
Comment (cyoon1729) [inline rl_algorithms/common/buffer/wrapper.py:22]: Change class name `PERWrapper` -> `PrioritizedBufferWrapper`?

File: rl_algorithms/common/buffer/wrapper.py [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
+    """Prioritized Experience Replay wrapper for ReplayBuffer.
```
Comment (MrSyee) [inline rl_algorithms/common/buffer/wrapper.py:23]: `Prioritized Experience Replay wrapper for ReplayBuffer -> Prioritized Experience Replay wrapper for Buffer`

File: rl_algorithms/common/buffer/wrapper.py (hunk 1 above)
Comment (jinPrelude) [reply rl_algorithms/common/buffer/wrapper.py:22]: OK!

File: rl_algorithms/common/abstract/buffer.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+from abc import ABC, abstractmethod
+from typing import Any, Tuple
+
+import numpy as np
+
+
+class BaseBuffer(ABC):
+    """Abstract Buffer used for replay buffer."""
+
+    @abstractmethod
```
Comment (MrSyee) [inline rl_algorithms/common/abstract/buffer.py:17]: should remove comments and add `pass`

File: rl_algorithms/common/buffer/wrapper.py (hunk 1 above)
Comment (jinPrelude) [reply rl_algorithms/common/buffer/wrapper.py]: resolved at 7348486.

File: rl_algorithms/common/buffer/wrapper.py (hunk 2 above)
Comment (jinPrelude) [reply rl_algorithms/common/buffer/wrapper.py]: resolved at 7348486.

File: rl_algorithms/common/abstract/buffer.py (hunk 3 above)
Comment (jinPrelude) [reply rl_algorithms/common/abstract/buffer.py:17]: resolved at 7348486.

**Pull request merged**